src/main/java/Todo.java

```diff
@@ -0,0 +1,9 @@
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }
+
+    public void printList(Task theTask,int index) {
+        System.out.println(index + ".[T][" + (theTask.getStatus()) + "] "+ theTask.getDescription());
```

Comment:
Can consider using printf for easier readability as the format is less confusing.